### ISSUE TITLE
Firefox 106.0: error messages and crashes

### ISSUE THREAD

**Issue by APMichael** [sandboxie-plus/Sandboxie#2356]
### Describe what you noticed and did

The new Firefox version 106.0 does not work well with Sandboxie.

On my Windows 10 21H2 (x64) I get these error messages every time I watch a video:
```
PID 2216: SBIE2336 Error in GUI server: [55 / C0000155]
PID 2216: SBIE2335 Initialization failed for process WerFault.exe [88 / 0]
```
(PID 2216 = SbieSvc.exe)

On a Windows 11 21H2 (x64) in a VM, I get the following error message:
```
WerFault.exe (3064): SBIE2203 Failed to communicate with Sandboxie Service: *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]
```
In addition, in my tests in the VM, Firefox crashed very often on  e.g. YouTube or Bing.

The issue was already mentioned for the beta of Firefox 106.0:
https://www.wilderssecurity.com/threads/sandboxie-plus-v1-3-4.447639/page-2#post-3108293
https://www.wilderssecurity.com/threads/sandboxie-plus-v1-3-4.447639/page-3#post-3109121

And has also already been confirmed for the release version:
https://www.wilderssecurity.com/threads/sandboxie-plus-v1-4-0-v1-4-1-v1-4-2.447865/page-3#post-3110628

### How often did you encounter it so far?

Each time a video is viewed.

### Affected program

Firefox 106.0

### Download link

https://ftp.mozilla.org/pub/firefox/releases/106.0/win64/

### Where is the program located?

The program is installed only outside the sandbox.

### Expected behavior

As with previous Firefox versions: no error messages and crashes.

### What is your Windows edition and version?

Windows 10/11 Pro, 21H2, 64-bit

### In which Windows account you have this problem?

User account with UAC protection set to Always notify.

### Please mention any installed security software

Microsoft Defender Antivirus

### What version of Sandboxie are you running?

1.4.2

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

**Comment by APMichael** [comment]
By the way, I have seen this report #2288 as well.

Uninstalling and reinstalling Firefox did not bring any improvement. Also, the tests are always done in a clean VM, so Sandboxie and Firefox have to be freshly installed.

**Comment by Seeadler1** [comment]
I can confirm since Firefox 106.0

**Comment by rugabunda** [comment]
Confirmed. Also crashes VideoDownloadhelper.

**Comment by APMichael** [comment]
> Updating to the latest v1.5.0 release fixes the issue.

I can also confirm that the provisional workaround works: no more error messages are triggered and I was not able to provoke a crash. 😃

**Comment by rugabunda** [comment]
No more crashing, or video watching. Updated to 1.5.0, videos no longer play in firefox in sandboxie.

**Comment by APMichael** [comment]
> No more crashing, or video watching. Updated to 1.5.0, videos no longer play in firefox in sandboxie.
> 
> UPDATE: actually, firefox wasn't loading in sandboxie, i had mistakenly removed firefox.exe the sandbox settings earlier. Now that I fixed that, I guess videos only work IN sandboxie.

However, this problem does not seem to be caused by Sandboxie, as this also happens to me from time to time, even when the Sandboxie service has been stopped manually. Since Firefox 104.0, there have been problems with YouTube, which seem to have increased again with 106.0. I noticed that it actually always happens when the "Cookie Consent Dialog" from YouTube was not displayed. Exiting and restarting Firefox always helped so far, or even briefly turning off uBlock, after which the dialog was displayed and the videos played again. The strange thing is that this problem has never occurred when Firefox was running in a sandbox.

**Comment by rugabunda** [comment]
> >  The strange thing is that this problem has never occurred when Firefox was running in a sandbox.

Thanks for unbreaking Firefox David!

**Comment by ganego** [comment]
@DavidXanatos I also just ran into this problem and tried updating Sandboxie. Problem: Auto updater gives 1.4.2 currently, which means normal users cannot update to a fixed version which I'd consider crucial in the case of a browser.

**Comment by CaliSunUSA** [comment]
Firefox updated to 106.0.3 and error messages returned.
So I looked at: https://github.com/sandboxie-plus/Sandboxie/releases/ and I see that new version is available (1.5.2)
When I try to install downloaded program, I get a message I have never seen:

![Sandboxie install error](https://user-images.githubusercontent.com/67880003/199129766-ea4ab483-f023-4c87-a2b3-75550279bcd5.png)
